servers/rendering/renderer_rd/shaders/screen_space_reflection.glsl

```diff
@@ -74,6 +74,14 @@ void main() {
 
 	vec4 normal_roughness = imageLoad(source_normal_roughness, ssC);
 	vec3 normal = normal_roughness.xyz * 2.0 - 1.0;
+	float roughness = normal_roughness.w;
+
+	if (roughness > 0.75) {
+		// Do not compute SSR for rough materials to improve performance at the cost of
+		// subtle artifacting.
+		return;
```

Comment:
We still need to write something to the output or else we will get artifacts in the subsequent passes. Just treat this as a no collision situation and write out black
```suggestion
		// Do not compute SSR for rough materials to improve performance at the cost of
		// subtle artifacting.
#ifdef MODE_ROUGH
		imageStore(blur_radius_image, ssC, vec4(0.0));
#endif
		imageStore(ssr_image, ssC, vec4(0.0));
		return;
```